.github/workflows/e2e.yaml

```diff
@@ -153,7 +154,7 @@ jobs:
       fail-fast: false
       matrix:
         kubernetes-version: # the GKE setup script ignores the patch version here and uses the latest, but we still include it for consistency with the other E2E jobs
-          - 'v1.24.2'
+          - 'v1.24.7'
```

Comment:
The comment above states that bumping this shouldn't make any difference 🤔

Reply:
> ... but we still include it for consistency with the other E2E jobs
Let's follow this for now.